src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -1120,85 +1402,55 @@ static SequenceComparison createSequenceComparison(final VariantContext variant,
                                                        final GencodeGtfTranscriptFeature transcript,
                                                        final List<? extends htsjdk.samtools.util.Locatable> exonPositionList,
                                                        final Map<String, MappedTranscriptIdInfo> transcriptIdMap,
-                                                       final ReferenceDataSource transcriptFastaReferenceDataSource) {
+                                                       final ReferenceDataSource transcriptFastaReferenceDataSource,
+                                                       final boolean processSequenceInformation) {
 
         final SequenceComparison sequenceComparison = new SequenceComparison();
 
         // Get the contig:
         sequenceComparison.setContig(variant.getContig());
 
         // Get the strand:
-        final Strand strand = Strand.decode( transcript.getGenomicStrand().toString() );
+        final Strand strand = Strand.decode(transcript.getGenomicStrand().toString());
```

Comment:
It seems unnecessary to go through string here.  It seems like it would be more efficient to have a direct conversion method.    It's also generally a sketchy idea to use toString to encode things like this, toString is generally meant as a human readable form, and we don't like to rely on it producing specific results.  Better to have an encode() method even if it matches the toString.

Reply:
While I was looking at this, I noticed some sort of strange patterns  GencodeGtfFeature.

ex:
```
 public enum GenomicPhase {
   
        public static GenomicPhase getEnum(final String s) {
            for( final GenomicPhase val : values() ) {
                if(val.serialized.equalsIgnoreCase(s)) {
                    return val;
                }
            }
            throw new IllegalArgumentException("Unexpected value: " + s);
        }
    }
```

You've added your own `getEnum(String s)` method.  There's a built in method valueOf() which does this already (although it doesn't ignore case).  It might make sense to normalize the case of everything before passing it in and use that.  Alternatively, if you're implementing this, you don't want to call `values()` repeatedly.  Every call to `values()` allocates a new array which is pretty expensive, you should cache it in a private variable and use that.

Reply:
That's fair.

In the future I'd like to refactor this to use the built-in `Strand` type, rather than the one I made for the `GencodeFuncotationFactory`.

Reply:
I made those enum patterns because I needed a serialization method where I could ignore the case of the enums before conversion.

I'll pull out the `values()` calls.  I'll need to think about case normalization.

Reply:
yeah, it makes sense if you need to ignore the case.